Changelog

```diff
@@ -50,6 +60,9 @@ Version 3.3, *2020-XX-XX*
       ContainerAudit module (#2029)
     * Fix a bug which prevented defining an SQL resolver without a password (#2030)
     * Fix missing "position" argument on event import with pi-manage (#2036)
+    * Fix timing issues in tests (#2041)
+    * Fix sorting token table by column (#2111)
+    * Fix errors, which occured due to the updating of the java script components (#2061 #2064 #2114 #2117 #2122 #2125 #2127)
```

Comment:
This is no fix. Since it was not broken in version 3.2.2. So we do not need ot mentioned this!

Reply:
done

Reply:
@laclaro Thanks! Please not that usually the reviewer should mark the comment as resolved.